cfut/condor.py

```diff
@@ -50,8 +50,8 @@ def submit_text(job):
     """Submits a Condor job represented as a job file string. Returns
     the cluster ID of the submitted job.
     """
-    out, _ = chcall('condor_submit -v', job)
-    jobid = re.search(r'Proc (\d+)\.0', out).group(1)
+    out, _ = chcall('condor_submit -v', job.encode('utf-8'))
+    jobid = re.search(rb'Proc (\d+)\.0', out).group(1)
```

Comment:
Testing the Condor code already revealed a couple of issues - the `subprocess` module works with bytes for stdin/stdout by default, rather than str.

Reply:
Oops; must be another undetected holdover from the Python 2 days.